### DISCUSSION TITLE
"Cold Caverns Syndrome" or why campaigns can be too long for newbies

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13188] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After experiencing the game with a crew of complete, 0 hours newbies, we ran into what I'd call "Cold Caverns Syndrome"

**Most new players will try to make the most out of the first biome without knowing how long the campaign really is.**

This leads to first campaigns stagnating for up to 20 hours in the first biome. 
Problem: there are 5 biomes, and this means the campaign wont be completed until at least 20 more hours. 
Those players tend to get bored and quit after spending so many hours in the cold caverns, having experienced only part of the content

The reasons: 

- Jovian Radiation being opt-in instead of opt-out. Players won't be pushed forward.
- "Long" missions in cold caverns. Mining missions tend to take the longest early on
- Presence of minerals + mineral scanner. Like mining above, it tends to slow down sessions.
- 1st reputation gate hints at grinding reputation, passage NPC can be hard to find

It's worth looking into nudging players forward out of cold caverns so they experience more of the content and don't accidentally spend most of their time in the tutorial area.
 
Introduce simple GoTo missions to the Cold Caverns pool exclusively, remove the mining missions?

### Reproduction steps

Find a group of friends who are completely new to the game.

### Version

v1.2.6.0 (Winter Update)

**Comment by ubertpendragon** [comment]
Should've opened a discussion instead 😬

**Comment by Ek-Videogames** [comment]
Additionally, the beginning of the campaign is when alot of "setup" occurs. 

Early players tend to spend alot more time at outposts on average: buying personal equipment, crafting resources, and submarine upgrades with the cash from missions. As well as taking the time at station to do menial tasks like organizing the ship's inventory, and waiting for crafting items to be produced (which can also be slow due to early game skills)

Compared to once you are in the endgame when you're usually just speedrunning to collect your missions and then buying a little bit of fuel/medical supplies/ammo when needed and not much else.

> - "Long" missions in cold caverns. Mining missions tend to take the longest early on
> - 1st reputation gate hints at grinding reputation, passage NPC can be hard to find

Yeah the "easy" missions are also many slow or otherwise uneventful which can increase the perception of tedium, 
and as for random events I see alot of people get bored and complain of the same old crawler groups & small mudraptor groups not knowing how much things ramp up later on with multiple molochs, hammerhead swarms, bonethreshers etc.

Also 'min-maxing' beacon missions being the only way to carry on having outposts into later biomes, (as well as being essentially a free 1k at the cost of some time and repair work, which is a pretty good reward in early-game) Beacons are cool but really slow down the pace of campaign same as mining missions.

Maybe reputation gate should be lowered or eliminated altogether, or perhaps hint at the opportunity to pay a bribe instead, in order to allow the player to push into the second biome without the issue of being turned away and forced to backtrack and grind missions. (A long standing concern ever since the 'reputation gates' were added to begin with) Building rep can be especially difficulty if unlucky with RNG and you are unable to chain together enough main-faction missions with high reputation rewards, and as an early player if its your first campaign you might not know it's potentially wiser to skip a potential separatist mission if you plan on grinding reputation with the coalition, and vice versa.

**Comment by Arctodites** [comment]
Jovian radiation as a mechanism to push players forward in the early biomes wouldn't even be as necessary if outposts only offered new missions rightward or parallel to the crew's current location, or better yet, if they offered missions, say,  one node behind and up to two nodes ahead.  It would be a kind of softlock -- a nudge -- on farming early missions.

I do like the idea of Jovian Radiation, but the prospect of having to deal with it for the whole game is annoying.  If it were something that players only encountered in the first two biomes (i.e. those closest to the irradiated surface), I think a lot more people would be happy to leave it enabled by default.  But as the biomes become richer and more complex in the later stages, I think people also want an opportunity to enjoy the sandbox aspects of Barotrauma, and not to feel pushed into such a linear progression.  When the long-term goals of a game feel too predetermined, it's easy for players to feel that they lack agency and to grow despondent or disinterested.  It's the narrative distinction between Fate and Destiny.

**Comment by duarm** [comment]
Another solution would be much shorter biomes, something like 3 missions deep to reach the next biome, and 9 missions total (3x3), without jovian radiation. You could then go to to the 3rd and 4th biome, and come to the 1st biome if needed, with some direct connections (from the 3rd to the 1st biome), not much time would be wasted. The last biome could be kept as is, as it's mechanically different (and thematically fitting).
